tests/pgsql/pgsql-bug-6983-ids/README.md

```diff
@@ -0,0 +1,12 @@
+# Description
+
+Tests that alerts for the pgsql app-proto will include pgsql app-proto metadata.
```

Comment:
Didn't alter this here as I have a feeling we may want to bring back this check to the tests.